lte/gateway/c/session_manager/SessionState.cpp

```diff
@@ -455,7 +495,21 @@ bool SessionState::apply_update_criteria(SessionStateUpdateCriteria& uc) {
       return false;
     }
   }
-
+  // Converged 5G rules to install
+  for (const auto& rule : uc.PdrList_to_install) {
+    if (!search_pdr(rule.pdr_id())) {
+      int32_t Pdr_index;
+      Pdr_index = pdr_index(rule.pdr_id());
+      if (Pdr_index != -1) {
+        // Update the previous one
```

Comment:
change the comment as `\\ Update the existing value`

Reply:
Changed.